src/view/app/Components/Layout/Query/QueryFormHead.tsx

```diff
@@ -51,8 +41,8 @@ const QueryFormHead: React.FC<QueryFormHeadProps> = ({
         }
     }, []);
 
-    const [anchorEl, setAnchorEl] = React.useState(null);
-    const [selectedOption, setSelectedOption] = React.useState('JSON');
+    const [anchorEl, setAnchorEl] = useState<HTMLElement>(null);
```

Comment:
Why are these variables here? They should belong to `QueryDropdownMenu` as these are not used elsewhere.